Installer/51-gcadapter.rules

```diff
@@ -0,0 +1 @@
+SUBSYSTEM=="usb", ENV{DEVTYPE}=="usb_device", ATTRS{idVendor}=="057e", ATTRS{idProduct}=="0337", MODE="0666"
```

Comment:
The package should probably `udevadm control --reload-rules` after uninstall/install as well.